ocrd_lib.sh

```diff
@@ -257,8 +257,12 @@ kitodo_production_task_action() {
       ;;
   esac
 
-  if test -n "$ACTIVEMQ" -a -n "$ACTIVEMQ_CLIENT" -a -n "$TASK_ID" -a -n "$ACTION"; then
-    java -Dlog4j2.configurationFile=$ACTIVEMQ_CLIENT_LOG4J2 -jar "$ACTIVEMQ_CLIENT" "tcp://$ACTIVEMQ?closeAsync=false" "TaskActionQueue" $TASK_ID "$MESSAGE" "$ACTION"
+  if test -n "$ACTIVEMQ" -a "$ACTIVEMQ" != ":" -a -n "$TASK_ID"; then
```

Comment:
Are you sure you want to allow passing an empty `$ACTION` here? (Don't know your client well enough, some programs can cope with empty CLI arguments.)

If not, I suggest keeping the extra condition (so nothing is attempted to send):

```suggestion
  if test -n "$ACTIVEMQ" -a "$ACTIVEMQ" != ":" -a -n "$TASK_ID" -a -n "$ACTION"; then
```

Reply:
Also, what happened to the older protocol where we used to send PROCESS_ID as message?